### ISSUE TITLE
Multiple contacts on a page

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#3127]
### As a resident, I want to see the location and hours information for multiple neighborhood centers on one page, so I can choose the one most convenient to me.

We need the ability to show multiple contacts on one page in Janis. See #3079 for more background. 

This is currently a blocker for publishing out Austin Public Health pages, so we're aiming for a smaller solution that solves our needs, doesn't make it worse for users, and isn't a huge dev lift. We'll wait on the new Place content type for a more ideal user experience that might require a bigger dev lift. 

### When and where this happens
Information pages only? (Right now, our need is just on information pages; but if contacts are working the same on all pages, maybe we should just keep it that way.)

### Janis
XD Prototype: https://xd.adobe.com/spec/f81c5f2d-3350-4720-79ff-cd73881cf143-8e2d/
[Mobile](https://xd.adobe.com/spec/f81c5f2d-3350-4720-79ff-cd73881cf143-8e2d/screen/bf44727f-d05d-43a2-a695-dea7e6bace1c/Template-multiple-locations-mobile), [desktop](https://xd.adobe.com/spec/f81c5f2d-3350-4720-79ff-cd73881cf143-8e2d/)

- [ ] Display order is now: LocationName, Address, Hours, Phone, Email. This is a universal change to the component. This groups like-information, and prioritizes physical addresses which makes the list of locations easier to scan by using the LocationName as a label.
- [ ] Spacing between location details is now `20px`
- [ ] Spacing between last location detail and the rule line is now `30px`
- [ ] Rule line is `2px`, `#DBDEE1`

moved collapsable hours to https://github.com/cityofaustin/techstack/issues/3149
```
- [ ] Icon for down-carrot is named "arrow_drop_down" in the Material icon font we have in Janis, or can be found here: https://material.io/resources/icons/?search=down&icon=arrow_drop_down&style=baseline
  - [ ] Default state should have the arrow facing down
  - [ ] Open state should rotate icon 90 degrees so that the arrow is facing up
- [ ] Hours are now collapsed behind a dropdown, and displays the current day and hours. I’m not sure what the right term is, but it should open in the way that an accordion does (where content below moves down, instead of overlaying like dropdowns typically are).
```

**Comment by desigonz** [comment]
@chasechenevert, can you add mobile mocks? Thanks!

**Comment by briaguya** [comment]
@desigonz @chasechenevert I've got a WIP branch up, need some feedback:

Info page: https://deploy-preview-550--janis.netlify.com/en/health-safety/health-records-certificates-2/birth-death-certificates/info-page-with-multiple-contacts/
Dept page: https://deploy-preview-550--janis.netlify.com/en/police-oversight
Service page: https://deploy-preview-550--janis.netlify.com/en/housing-utilities/recycling-trash-and-compost/household-waste/bulk-item-pickup/

need to figure out logic around what to put as the title (it used to be "contact details" but now it's "locations")

I also haven't implemented the accordion for hours, @desigonz I think we should split that out into a separate issue, as it could easily balloon in scope.

**Comment by desigonz** [comment]
@chasechenevert @briaguya Looks like changing the title to "Locations" changes it across the board, which I don't think totally makes sense for single instances or for something like [Get your bulk items collected](https://deploy-preview-550--janis.netlify.com/en/housing-utilities/recycling-trash-and-compost/household-waste/bulk-item-pickup/). I'm in favor of reverting back to "Contact information" and then making a new issue to think this through.

**Comment by desigonz** [comment]
Another note: I believe we're dropping the location name in the location section.
![image](https://user-images.githubusercontent.com/5839508/66403210-cd9feb80-e9ab-11e9-895b-38cb237687d4.png)

Otherwise it's looking good.

**Comment by briaguya** [comment]
@desigonz updated it to not have the location name in the address portion, as well as set the main heading back to "Contact information"

**Comment by chasechenevert** [comment]
Spec updates: 
- added mobile
- reverted "Locations" to "Contact information"

**Comment by briaguya** [comment]
@chasechenevert I currently have it so the rule line doesn't exist when we only have a single contact, should I add it for that scenario?

**Comment by chasechenevert** [comment]
Agreed, Brian. I'll remove it from the spec

**Comment by briaguya** [comment]
@chasechenevert the naive implementation I have at the moment just adds a rule line above every contact that isn't the first. This means that:
* If there is one contact, there is no rule line
* For the last contact, there is no rule line at the bottom

Looking to confirm if that's an ok implementation or if I should add a little bit of extra logic for having the bottom rule line

**Comment by chasechenevert** [comment]
That sounds great to me, Brian!

**Comment by desigonz** [comment]
qa comp[lete
